### PR TITLE
add --include-tpcc-fail flag to include TPCC .bak result files in CSV

### PR DESCRIPTION
**Pull request by srosenberg**
revert cpuCSVHeader

### CODE REVIEW

File: cmd/analyze.go [hunk 1]
```diff
@@ -862,8 +865,15 @@ func (t *tpccAnalyzer) analyzeTPCC(cloud CloudDetails, machineType string) error
 	if err != nil {
 		return err
 	}
+	runs := goodRuns
```
Comment (ZhouXing19) [inline cmd/analyze.go:868]: I just realize `runs` seems redundant to `resultFiles` below, right? Maybe we can change the line above to 
```
glob := path.Join(cloud.LogDir(), FormatMachineType(machineType), "tpcc-results.*/success")
```
and only take care of the `includeTpccFail` condition  in the `for _, r := range runs` loop.

**Review by ZhouXing19 — CHANGES_REQUESTED** [review]

File: cmd/analyze.go (hunk 1 above)
Comment (srosenberg) [reply cmd/analyze.go:868]: Nice catch!

Comment (srosenberg) [reply cmd/analyze.go]: Updated; sorry for force pushing, mixing git workflows should be discouraged :)

**Review by ZhouXing19 — APPROVED** [review]
LGTM, thanks!

File: cmd/analyze.go (hunk 1 above)
Comment (srosenberg) [reply cmd/analyze.go]: The diff after amend is a nice feature: https://github.com/cockroachlabs/cloud-report/compare/7c7c3818febbdd6d07b3014d0319c4c30bd8660b..e5acef3e836e8cb724e72d66bd489dc90baf02b5

Comment (ZhouXing19) [reply cmd/analyze.go]: Re-LGTM!

**Pull request closed without merging**